### PR TITLE
{bio}[foss/2017b] Bowtie2 v2.3.4.1

### PR DESCRIPTION
**Pull request by murfreesboro**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/3831a57a8dc91cfb3481dd47a4e5dbdf for a full test report.

File: easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.4.1-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True, 'cstd': 'c++11'}
+
+source_urls = [('http://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download')]
+sources = ['%(namelower)s-%(version)s-source.zip']
+checksums = ['a1efef603b91ecc11cfdb822087ae00ecf2dd922e03c85eea1ed7f8230c119dc']
+
+dependencies = [('tbb', '2018_U2')]
```
Comment (verdurin) [inline easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.4.1-foss-2017b.eb]: @murfreesboro - is there a specific reason why you removed the `zlib` dependency?

Comment (murfreesboro) [reply easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.4.1-foss-2017b.eb]: Okay, according to the manual it seems zlib is needed for the Bowtie2. However this file is modified from the Bowtie2-2.3.4.1-intel-2017b.eb where there is no zlib dependency. I will updated the dependencies section to include the zlib.

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ce37fe43851b5059fb0b0ba09064c7bb for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/903c3ef5e7ed634d4b6b04e37ee52a84 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/cdf06ee6f61c3cc12cdb19ef9a33bb0e for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @murfreesboro!

**Pull request merged**